src/controller/MovieController.php

```diff
@@ -26,10 +26,25 @@ public function getAllActiveMovies(): array {
         if (isset($movies['error']) && $movies['error']) {
             return ['errorMessage' => $movies['message']];
         }
-        
         return $movies;
     }
 
+    public function isArchived(int $movieId): bool {

```

Comment:
can't this be determined from the "Movie" entity? Don't we have 'isArchived' or 'archived' boolean property on the "Movie" entity?

Reply:
And it also seems like this method is unused, has no reference anywhere